src/OrangePaymentSlipData.php

```diff
@@ -54,6 +55,13 @@ class OrangePaymentSlipData extends PaymentSlipData
      * @var string
      */
     protected $bankingCustomerId = '';
+    
+    /**
+     * The banking customer ID expected length
+     *
+     * @var integer
+     */
+    protected $bankingCustomerIdLength = 0;
```

Comment:
Setting it to zero seems to be a bit silly. Either unset or a reasonable default.